src/modules/QtGraphicalEffects/RectangularGlow.js

```diff
@@ -41,7 +31,7 @@ QmlWeb.registerQmlType({
     this.spreadChanged.connect(this, this.$onSpreadChanged);
   }
   $onColorChanged(newVal) {
-    this.impl.style.backgroundColor = new QmlWeb.QColor(newVal);
+    this.impl.style.backgroundColor = newVal;
```

Comment:
Yes, this should be converted to `newVal.$css` in some of the PRs, depending on which one of those would be merged first. I will handle that while landing.